src/test/java/seedu/address/testutil/PersonBuilder.java

```diff
@@ -115,7 +115,7 @@ public PersonBuilder withEmergencyContact(String phone) {
      * Sets the {@code Level} of the {@code Person} that we are building.
      */
     public PersonBuilder withLevel(String level) {
-        this.level = !level.isEmpty() ? new Level(level) : new Level("NONE");
+        this.level = level.isEmpty() ? null : new Level(level);
```

Comment:
shld it be Level("NONE NONE") or null? Imo shld be the former since that is what Level is set to if a person is added without a level